src/function/algebra/rationalize.js

```diff
@@ -177,6 +178,8 @@ export const createRationalize = /* #__PURE__ */ factory(name, dependencies, ({
       const setRules = rulesRationalize() // Rules for change polynomial in near canonical form
       const polyRet = polynomial(expr, scope, true, setRules.firstRules) // Check if expression is a rationalizable polynomial
       const nVars = polyRet.variables.length
+      const optsNoEF = { exactFractions: false }
```

Comment:
Small thing: maybe rename this to `optsNoExactFractions`? Too much abbreviations make it harder to understand the code 😅.

Reply:
Check.